lib/howl/ui/style.moon

```diff
@@ -234,6 +234,19 @@ set_for_theme = (theme) ->
   for name in pairs styles
     define_extended name if is_extended name
 
+ -- set status styles
```

Comment:
Yes, please do, it just adds a special case for code that was previous pretty generic. Besides, the duplication in the themes is not a big issue in my opinion. In the future we can get rid of the GtkLabel instead and just use a normal editing view which can use all the normal styles and Howl markup instead.